lib/process-services/src/lib/task-list/components/task-form/task-form.component.spec.ts

```diff
@@ -161,6 +163,16 @@ describe('TaskFormComponent', () => {
             expect(errorSpy).toHaveBeenCalled();
         });
 
+        it('Should be able to disable complete button in case task shared with candidates and initiatorCanCompleteTask set to true', async () => {
```

Comment:
`Should be able to disable complete button`
what does it mean? 
Should be able to complete a candidate task if the user is the initiator? 
Can we make sure we cover all the combinations?
Task with assignee started by the initiator  (should be able to complete)
Task with assignee NOT started by the initiator (should NOT be able to complete)
Task with candidate users started by the initiator (should be able to claim and complete)
Task with candidate users NOT  started by the initiator (should NOT be able to claim and complete)

Reply:
covered above scenarios please review thanks https://github.com/Alfresco/alfresco-ng2-components/pull/5780/files#diff-33411bc1a3aec7d706037f6f3a5cec1fR487